docs/wasmer-js/module-transformation.md

```diff
@@ -41,16 +39,41 @@ Ok, back in reality...
 
 ## The [`clock_time_get`](https://raw.githubusercontent.com/wasmerio/docs.wasmer.io/master/docs/wasmer-js/wasm_lib/clock_time_get.wat) WebAssembly Module
 
-In order to understand whether or not this module needs transformation, we need to take a look inside the WebAssembly module.
+In order to understand whether or not this module needs transformation, we need to take a look at the WebAssembly module.
 
-> ### ASIDE
-> We make no attempt to teach you WebAssembly here!
->
-> If you want to know about the inner workings of a WebAssembly module, then please visit the [WebAssembly.org](https://webassembly.org) website and read the documentation there.
->
-> We now continue with your scheduled program...
+In the Transforming WASI Modules examples, there is an example [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) module, in which is uses the [clock_time_get WASI binding from the wasi_unstable phase](https://github.com/AssemblyScript/assemblyscript/blob/899e97ab28a857da07a533551eb937d771677f93/std/assembly/bindings/wasi_unstable.ts#L36). For example:
```

Comment:
A couple of somewhat pedantic points here...

* Grammatical use of the passive voice.  The above sentence should read   
    "...example AssemblyScript module, in which the clock_time_get WASI binding from the wasi_unstable phase ***is used***"  
    😀

* Use of previously un-explained terminology.  
    Is it reasonable for a non-expert to understand what is meant by the term "*wasi_unstable phase*"?  
    May I suggest that this term is either not used, or given a brief explanation

Thanks